src/reanimated2/frameCallback/FrameCallbackRegistryUI.ts

```diff
@@ -55,6 +99,9 @@ export const prepareUIRegistry = runOnUI(() => {
         this.frameCallbackActive.add(frameCallbackId);
         this.runCallbacks();
       } else {
+        const callback = this.frameCallbackRegistry.get(frameCallbackId)!;
```

Comment:
It looks like you're using different names for the same thing:
* `key`
* `callbackId`
* `frameCallbackId`.

You probably need to choose one.

Reply:
I went with `callbackId`